src/AutoMerge/guidelines.jl

```diff
@@ -424,7 +424,8 @@ function meets_repo_url_requirement(pkg::String; registry_head::String)
 end
 
 function _invalid_sequential_version(reason::AbstractString)
-    return false, "Does not meet sequential version number guideline: $reason", :invalid
+    return false, "Does not meet sequential version number guideline: $reason\n
+    If this was intentional, please leave a comment saying so. Otherwise please correct the version number in your package and re-trigger registration.", :invalid
```

Comment:
One last tweak here; I think we shouldn't have a `\n` and can just let it break the text as it natural.
```suggestion
    return false, "Does not meet sequential version number guideline: $(reason). If this was intentional, please leave a comment saying so. Otherwise please correct the version number in your package and re-trigger registration.", :invalid
```

Reply:
OK, as you wish. The important thing is the message itself. The format isn't that important to me.